src/journal/journald-server.c

```diff
@@ -2035,6 +2078,92 @@ static int server_open_varlink(Server *s) {
         return 0;
 }
 
+static bool server_is_idle(Server *s) {
+        assert(s);
+
+        /* The server for the main namespace is never idle */
+        if (!s->namespace)
+                return false;
+
+        /* If a retention maximum is set we need to be running to enforce it */
+        if (s->max_retention_usec > 0)
```

Comment:
We could have max_retension_sec < idle_time, so exit on idle would still be possible.

Reply:
true